File: 프로그래머스/0/181928. 이어 붙인 수/이어 붙인 수.js
function solution(num_list) {
    let ret = 0;
    
    const odd = num_list.filter((num)=> num % 2 !== 0);
    const oddConverted = odd.map((el)=> String(el)).join("");
    
    const even = num_list.filter((num)=> num % 2 === 0);
    const evenConverted = even.map((el)=> String(el)).join("");
    
    
    return Number(oddConverted) + Number(evenConverted);
}

File: 프로그래머스/0/181928. 이어 붙인 수/이어 붙인 수.py
def solution(num_list):
    odd = [i for i in num_list if i % 2 != 0]
    even = [i for i in num_list if i % 2 == 0]
    return int("".join([str(i) for i in odd])) + int("".join([str(i) for i in even]))